src/wasm.h

```diff
@@ -91,6 +91,11 @@ struct Name : public cashew::IString {
 // An index in a wasm module
 typedef uint32_t Index;
 
+inline Index toIndex(size_t index, size_t max) {
+	//assert(index < max);
```

Comment:
I'm being stupid. I removed the assert to test and then forgot about it. This explains why mapLocals didn't assert. I'll uncomment.
 

    On Thursday, May 19, 2016 8:01 PM, Alon Zakai <notifications@github.com> wrote:
 

 In src/wasm.h:> @@ -91,6 +91,11 @@ struct Name : public cashew::IString {
>  // An index in a wasm module
>  typedef uint32_t Index;
>  
> +inline Index toIndex(size_t index, size_t max) {
> +	//assert(index < max);
indentation looks wrong here—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub